helm/polaris/LICENSE

```diff
@@ -0,0 +1,13 @@
+  Copyright (c) 2024 Snowflake Computing Inc.
```

Comment:
Technically this is a copyright statement.

Do we need a separate LICENSE file here in addition to what is in the repo root?

Reply:
You are right there too. Yes, I was copied the license header (copyright statement) into this file. I had updated this one as well with the content of https://github.com/polaris-catalog/polaris/blob/main/LICENSE.

Now to answer your question. Assuming we will be publishing this to [Artifact Hub](https://artifacthub.io/) later on. The LICENSE file will be missing in that case with following sample command:
```
helm package polaris/
```

Or we can update our public script to copy the one from root dir then discard it after build. @dimas-b / @flyrain what do you think?

Reply:
Having a license file in helm charts makes sense to me. I was just trying to avoid duplication... Would a symlink work instead of a copy here?

Reply:
Very good idea. Updated.